topics/introduction/tutorials/galaxy-onDockstore/galaxy-onDockstore.html

```diff
@@ -0,0 +1,221 @@
+---
+layout: Dockstore_tutorial
+logo: "GTN"
+
+title: "Galaxy workflows in Dockstore"
+objectives:
+  - "What is Dockstore?"
+  - "Why use Dockstore?"
+  - "How to find workflows on Dockstore?"
+  - "How to launch workflows from Dockstore?"
+  - "Getting started with Galaxy workflows on Dockstore"
+  - "Register your Galaxy workflow on Dockstore"
+subtopic: Dockstore
+contributors:
+  - avani-k
+  - bethsheets
+  - coverbeck
+  - afgane
+---
+
+# What is Dockstore?
+
+---
+
+## Dockstore: “an app store for bioinformatics”
+
+.pull-left[
+
+- Open-source platform for sharing reusable bioinformatics workflows
+
+- Easy and reliable way to find and use relevant workflows as well as publish your own workflows
+
+- Dockstore integrates source control repositories like GitHub, Bitbucket, and GitLab for version control and automatic updates.
+
+- Integration with several cloud environments such as the usegalaxy.org, Galaxy Project's public server provides a succinct infrastructure to host high quality workflows
+
+]
+
+.pull-right[
+
+.image-40[
+  ![Structure of Dockstore]({{ site.baseurl }}/assets/images/Dockstore-Schematic.png)
+]
+
+[dockstore.org](https://dockstore.org/)
+
+]
+
+---
+
+# What is Dockstore?
+
+---
+
+## Why use Dockstore?
+
+Dockstore follows the FAIR Guiding Principles (Findable, Accessible, Interoperable and Reusable).
+
+## Findable
+
+- Users are strongly suggested to include robust metadata and human readable instructions. Dockstore parses provided metadata and allows it to be searchable.
```

Comment:
I'm a bit worried it might be a lot of text per slide, do you have maybe more screenshots you'd like to include?

Also there's the option to put a lot of the text into 'speaker notes' and make the slides focus on the bullet points. And then if you have good speaker notes, you can enable the automated videos or so? https://training.galaxyproject.org/training-material/topics/contributing/tutorials/slides-with-video/tutorial.html

Reply:
I don't have screenshots ready for use but let me mull over it a bit. Also, I'll look into adding speaker notes.

Reply:
no worries if out of scope and you need this for a workshop, just seemed like a potential place that could be 'too much text' for some readers.

Reply:
I haven't added speaker notes as of yet. Might be a good addition for a future PR.